PWGHF/TableProducer/candidateCreator2Prong.cxx

```diff
@@ -677,6 +677,7 @@ struct HfCandidateCreator2ProngExpressions {
 
   // Configuration
   o2::framework::Configurable<bool> rejectBackground{"rejectBackground", true, "Reject particles from background events"};
+  o2::framework::Configurable<bool> matchKinkedDecayTopology{"matchKinkedDecayTopology", true, "Match also candidates with tracks that decay with kinked topology"};
```

Comment:
Let's put it false not to change the default behaviour compared to the previous version 
```suggestion
  o2::framework::Configurable<bool> matchKinkedDecayTopology{"matchKinkedDecayTopology", false, "Match also candidates with tracks that decay with kinked topology"};
```